modules/static/opentelemetry/opentelemetry/initref

```diff
@@ -1 +1 @@
-4a2aefa60114914fa7f2acd40ab376c9fc1f1e19
+v0.8.0
```

Comment:
This is the the closest tag:

```
➜  opentelemetry-proto git:(4a2aefa) ✗ git describe --all 4a2aefa60114914fa7f2acd40ab376c9fc1f1e19
tags/v0.8.0-1-g4a2aefa
```

Here are all the releases: https://github.com/open-telemetry/opentelemetry-proto/releases